### PR TITLE
Bump expo version

### PR DESCRIPTION
**Pull request by bonesyblue**
Bumps expo SDK version to v45

* Replace [@react-native-clipboard/clipboard](https://github.com/react-native-clipboard/clipboard) with [expo-clipboard](https://docs.expo.dev/versions/latest/sdk/clipboard/)
* Replace [@use-expo/font](https://github.com/bycedric/use-expo) with [@expo-google-fonts](https://github.com/expo/google-fonts)
* Replace [react-native-appearance](https://github.com/expo/react-native-appearance) with [Appearance API](https://reactnative.dev/docs/appearance)

### CODE REVIEW

File: app/App.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
+// eslint-disable-next-line camelcase
+import { Poppins_400Regular, useFonts } from '@expo-google-fonts/poppins';
```
Comment (digorath) [inline app/App.tsx:2]: does this mean, that we no longer need the font in the assets?

**Review by digorath — COMMENTED** [review]
Just one question. Rest looks good to me :)

File: app/App.tsx (hunk 1 above)
Comment (bonesyblue) [reply app/App.tsx:2]: Good catch! 🎾 I'll have a look later at whether this can also be removed 💯

Comment (digorath) [reply app/App.tsx:2]: I created issue #46 for it, so that it no longer blocks this PR

**Review by digorath — APPROVED** [review]

**Pull request merged**